docs/build/chain-upgrades/testnet.mdx

```diff
@@ -19,3 +19,8 @@ sidebar_position: 2
 | Tag                                                                             | Name       | Start Height                                                       | Notes 
 |---------------------------------------------------------------------------------|------------|--------------------------------------------------------------------|----------------|
 | [`v9.0.0-rc.3`](https://github.com/noble-assets/noble/releases/tag/v9.0.0-rc.3) |            | [24114000](https://www.mintscan.io/noble-testnet/block/24114000)   |
+| [`v10.0.0-rc.3`](https://github.com/noble-assets/noble/releases/tag/v10.0.0-rc.3) |            | [31525000](https://www.mintscan.io/noble-testnet/block/31525000)   | must set `--halt-height 33800000` to upgrade to next version
```

Comment:
I think we are missing one. We need to upgrade to `v10.0.0-rc.2` here first at height `30385000`. I remember this upgrade required some troubleshooting.

Upgrade scheduled on `v9.0.0-rc.3` expects an upgrade name of of [v10.0.0-rc1](https://github.com/noble-assets/noble/blob/v10.0.0-rc.2/upgrade/constants.go)
Then we can go ahead and upgrade to [v10.0.0-rc.3](https://github.com/noble-assets/noble/blob/573189a814f7b07682f5edfa7a3d7f768f366cae/upgrade/constants.go#L20)

Reply:
https://www.mintscan.io/noble-testnet/tx/7E1829BB1550F52395EA9A46FCCE98470EFF691FF47E3DFDB2D3AA1066686F4A?height=30191730